Exiled.API/Features/Items/Jailbird.cs

```diff
@@ -75,6 +75,16 @@ public float ChargeDamage
         /// </summary>
         [EProperty(category: nameof(Jailbird))]
         public float FlashDuration
+        {
+            get => Base._hitreg._flashedDuration;
+            set => Base._hitreg._flashedDuration = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the amount of time in seconds that the <see cref="CustomPlayerEffects.Concussed"/> effect will be applied on being hit.
+        /// </summary>
+        [EProperty(category: nameof(ConcussionDuration))]
```

Comment:
```suggestion
        [EProperty(category: nameof(Jailbird))]
```

I'm retarded, revert it lmao